tests/expireat_test.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	_ "github.com/dicedb/dice/testutils"
+	"gotest.tools/v3/assert"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestEvalEXPIREAT(t *testing.T) {
+	conn := getLocalConnection()
+
+	expireInSeconds := int64(10) // Set expiration time to 10 seconds later
+
+	testCases := []struct {
+		name     string
+		commands []string
+		expected []interface{}
+	}{
+		{
```

Comment:
Please add test cases for the following scenarios:
1. EXPIREAT for non-existent key.
2. EXPIREAT time which is in the past (key should be deleted)
3. EXPIREAT with invalid syntax (no timeout provided)

Reply:
Please fix this.